neon/data-collection/data-streams/index.md

```diff
@@ -84,20 +84,22 @@ Audio recording is disabled in the Neon Companion app by default and can be enab
 <Badge>Real-time</Badge><Badge>Pupil Cloud</Badge><Badge>Neon Player</Badge>
 The Neon module is equipped with a 9-DoF [inertial measurement unit](https://invensense.tdk.com/products/motion-tracking/9-axis/icm-20948/) (IMU) featuring an accelerometer, gyroscope, and magnetometer. The accelerometer and gyroscope measure linear acceleration and angular velocity, respectively, and are provided as raw values.
 
-A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation relative to magnetic north and gravity as a quaternion. Note that in order to obtain precise absolute yaw readings, the magnetometer needs to be [calibrated](/data-collection/calibrating-the-imu/).
+A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation relative to magnetic north (positive world y-axis), gravity (negative world z-axis), and a rightward pointing vector (positive world x-axis) as a quaternion. We refer to this as the world coordinate system. It is important to note that this is not the same as the local IMU coordinate system.
```

Comment:
```suggestion
A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation in the world as a quaternion. The world coordinate system is defined such that magnetic north corresponds to the positive y-axis, gravity corresponds to the negative z-axis, and a rightward pointing vector to these corresponds to the positive x-axis.
```
I propose this slightly altered version. IMO a formulation that says "The world coordinate system is [...]" is easier to understand.

Reply:
Yes, thanks!